azavea/raster-foundry
# Add PostgreSQL 9.6 support

## Overview

Upgrade the PostGIS/PostgreSQL database image to version 2.3-postgres9.6-slim. This container image inherits from the official `postgres` container image and installs PostGIS package dependencies.

Fixes https://github.com/azavea/raster-foundry/issues/2183
Connects #2125 
See: https://github.com/azavea/docker-postgis

### Notes

Switching to this image will wipe your current development database. I tested the `load_development_data` script, and it worked fine for the things that I tested. If you really want to keep your previous database state, please back it up prior to review with `pg_dump`.

## Testing Instructions

To make sure things were working properly, I destroyed my local development environment and rebuilt it from scratch. That level of testing shouldn't be necessary here though. 

Bring up your development environment, execute `bootstrap` and `setup`. Then, execute `server` and make sure that all of the services come up properly.

Also, use `docker-compose ps` from the root of the development environment to ensure that the new PostGIS/PostgreSQL image is being used.
